nemo_curator/utils/distributed_utils.py

```diff
@@ -218,6 +226,11 @@ def read_single_partition(
     else:
         raise RuntimeError("Could not read data, please check file type")
 
+    if input_meta:
```

Comment:
Since this argument is only valid for `jsonl` formats, would it make sense to move it to the section above where `filetype=="jsonl"`? We can raise a warning if the user specifies this for other filetypes saying it'll not do anything.

Reply:
Great comment. Will do it right now.